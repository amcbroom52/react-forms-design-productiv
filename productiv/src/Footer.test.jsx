import { describe, it, expect } from "vitest";

import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer";

describe("Footer component", function () {
    it("renders without crashing", function () {
        render(<Footer />);
    });

    it("matches snapshot", function () {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    })

    it("contains Todo values", function() {
        const { container } = render(<Footer />);
        expect(container).toContainHTML("Prødutïv is copyright ©2020 by Flüffy Data Enterprises, Inc.");
    })
});