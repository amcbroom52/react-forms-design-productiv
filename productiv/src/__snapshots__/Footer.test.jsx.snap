// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="App-footer mt-5 text-muted"
  >
    <small>
      Prødutïv is copyright ©2020 by Flüffy Data Enterprises, Inc.
    </small>
  </footer>
</DocumentFragment>
`;
